testing/ostest/prioinherit.c

```diff
@@ -530,6 +530,8 @@ void priority_inheritance(void)
   g_medpri = my_pri - 1;
 
   sem_init(&g_sem, 0, NLOWPRI_THREADS);
+  sem_setprotocol(&g_sem, SEM_PRIO_INHERIT);
```

Comment:
why not change to nxmutex?

Reply:
ostest is app,nxmutex is kernal,so we need add mutex for app?

Reply:
since this is used to test the api usage, it's better to call the core api directly instead of wrapper.